nix/overlays.nix

```diff
@@ -25,17 +25,21 @@ self: final: prev: {
             # use the source of the git repo
             src = ./..;
             disabled = false;
+            postPatch = ''
+              substituteInPlace libqtile/pangocffi.py \
+              	--replace 'load_library("gobject-2.0")' 'ffi.dlopen("${prev.glib.out}/lib/libgobject-2.0.so.0")' \
```

Comment:
Btw we could use `${lib.getLib pkg}/libpkg.so` instead of `${pkgs.out}/lib/libpkg.so`